integration_tests/features/python_nodes.feature

```diff
@@ -45,3 +45,14 @@ Feature: Python nodes
       fal flow run --profiles-dir $profilesDir --project-dir $baseDir --select no_write_model
       """
     Then it throws an exception AssertionError with message 'There must be at least one write_to_model call in the Python Model'
+
+  Scenario: Broken DBT model leads to corruption of the dependant Python model
+    Given the project 014_broken_dbt_models
+    When the following command is invoked:
+      """
+      fal flow run --profiles-dir $profilesDir --project-dir $baseDir --select get_data regular_model
+      """
+    Then the following models are calculated:
+      | regular_model | get_data.py |
+    And the following scripts are ran:
+      | regular_model.post_hook.py |
```

Comment:
You could add a "and the exception X is thrown" which right now is a no-op. But we will make work eventually.

Reply:
This shouldn't supposed to a throw an exception.

Reply:
I thought we were running a failing run? But now I see we are doing `--select get_data regular_model`. Why aren't we running `process_data`? What is this actually testing?

Reply:
This test is ensuring that when there is a problematic model down in the chain (it doesn't really matter if it is selected or not, since this happens way before we exclude stuff in the graph) it doesn't affect the execution of others (just like how it doesn't effect it for DBT itself).

Reply:
But then it is not about the model being broken. It's just about the model not overwriting the dependencies as dbt models when they are actually fal models, right?

Reply:
> But then it is not about the model being broken. It's just about the model not overwriting the dependencies as dbt models when they are actually fal models, right?

Weirdly no. Somehow the model's broken status down in the chain affects the model listing / dependency order.